src/plugins/WebIndexPlugin.ts

```diff
@@ -18,7 +18,7 @@ export interface IndexPluginOptions {
     template?: string;
     templateString?: string;
     appendBundles?: boolean;
-    async?: boolean;
```

Comment:
Can we avoid breaking changes? Please keep it there and just update your object accordingly

Reply:
@nchanged I got your point!

Is it ok to keep async property and add scriptAttributes as collector of other attributes?

Or is to be preferred a defer (boolean) property instead of scriptAttributes?

Let me know